src/account.js

```diff
@@ -1,5 +1,5 @@
-const nacl = require('./nacl/naclWrappers');
-const address = require('./encoding/address');
+import nacl from './nacl/naclWrappers';
```

Comment:
I ran a [common-js to esmodule codemod](https://github.com/5to6/5to6-codemod) to convert all of these.